cypress/support/pages/admin/pageAdministration.js

```diff
@@ -22,7 +22,7 @@ class Management {
 	static #videoconferenceToggleSwitch = ".videoconference-switch";
 	static #saveGeneralSettingsButton = ".my-5";
 	static #tableContents = '[data-testid="table-data-body"]';
-	static #manageSchoolCard = '[data-testid="school_administration_card"]';
+	static #manageSchoolPage = '[data-testid="Schule"]';
```

Comment:
You can use static #schoolAdministrationNavigationButton = '[data-testid="Schule"]'; in line 34

Reply:
I can update it to 'schoolAdministrationNavigationSubMenu', as its not a button I would say.